### PR TITLE
Fit and finish on the rest docs of Azure Cognitive Search [2019-10-01-Preview]

### PR DESCRIPTION
**Pull request by jqhuangonearth**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/search.json [hunk 1]
```diff
@@ -1070,25 +1072,28 @@
...
     },
     "IpRule": {
       "properties": {
         "value": {
           "type": "string",
           "description": "Value corresponding to a single IPv4 address (eg., 123.1.2.3) or an IP range in CIDR format (eg., 123.1.2.3/24) to be allowed."
         }
-      }
+      },
+      "description": "The IP restriction rule of the Cognitive Search service."
```
Comment (abhi1509) [inline specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/search.json]: Azure Cognitive Search for consistency

**Comment by tchristiani** [comment]
@fengzhou-msft please work with @huangbolun to reassign the review request to another reviewer. @brjohnstmsft is saying he is not the correct contact for review on this.

**Comment by huangbolun** [comment]
@fengzhou-msft , could you help review and merge the change? We are catching up with an urgent deadline on Jan 10 for publishing the swagger docs. Let me know if you have questions. Much appreciated.

**Review by abhi1509 — APPROVED** [review]

**Comment by abhi1509** [comment]
#sign-off

**Comment by abhi1509** [comment]
> @fengzhou-msft please work with @huangbolun to reassign the review request to another reviewer. @brjohnstmsft is saying he is not the correct contact for review on this.

Please add me @abhi1509 as reviewer.

**Review by arv100kri — APPROVED** [review]

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**